apps/nova/src/components/TutorSettings/TutorPersonalInfoSettings.tsx

```diff
@@ -0,0 +1,183 @@
+import { useUpdateTutorTopics } from "@litespace/headless/user";
+import { useToast } from "@litespace/luna/Toast";
+import { useValidateUserName } from "@litespace/luna/hooks/validation";
+import { UseFormReturn } from "react-hook-form";
+import { ITutorSettingsForm } from "@/components/TutorSettings/common";
+import { useInvalidateQuery } from "@litespace/headless/query";
+import { QueryKey } from "@litespace/headless/constants";
+
+export const TutorPersonalInfoSettings: React.FC<{
+  tutor: {
+    video: string | null;
+    name: string | null;
+    bio: string | null;
+    about: string | null;
+    topics: { id: number; label: string }[];
+  };
+  form: UseFormReturn<ITutorSettingsForm, unknown, undefined>;
+}> = ({ tutor, form }) => {
+  const intl = useFormatMessage();
+  const toast = useToast();
+  const [topicsDialogOpen, setTopicsDialogOpen] = useState<boolean>(false);
+
+  const closeDialog = useCallback(() => setTopicsDialogOpen(false), []);
+  const openDialog = useCallback(() => setTopicsDialogOpen(true), []);
+
+  const validateUserName = useValidateUserName();
+  const invalidateQuery = useInvalidateQuery();
+
+  const topics = useTopics({});
+
+  const allTopics = useMemo(() => {
+    if (!topics.query.data) return [];
+    return topics.query.data.list.map((topic) => ({
+      id: topic.id,
+      label: topic.name.ar,
+    }));
+  }, [topics.query.data]);
+
+  const userTopicsIds = useMemo(() => {
+    return tutor.topics.map((topic) => topic.id);
+  }, [tutor.topics]);
+
+  const onTopicChangeSuccess = useCallback(() => {
+    invalidateQuery([QueryKey.FindUserTopics]);
+    closeDialog();
+  }, [invalidateQuery, closeDialog]);
+
+  const onTopicChangeError = useCallback(() => {
+    toast.error({
+      title: intl("tutor-settings.personal-info.update-topics-error"),
+    });
+  }, [intl, toast]);
+
+  const updateTopics = useUpdateTutorTopics({
+    onSuccess: onTopicChangeSuccess,
+    onError: onTopicChangeError,
+  });
+
+  const changeTopics = useCallback(
+    (topicsIds: number[]) => {
+      const addTopics: number[] = topicsIds.filter(
+        (topic) => !userTopicsIds.includes(topic)
+      );
+
+      const removeTopics: number[] = userTopicsIds.filter(
+        (topic) => !topicsIds.includes(topic)
+      );
+
+      updateTopics.mutate({
+        payload: {
+          removeTopics,
+          addTopics,
+        },
+      });
+    },
+    [updateTopics, userTopicsIds]
+  );
+
+  return (
+    <div className="flex flex-col gap-6 p-10">
+      <Typography
+        element="subtitle-1"
+        weight="bold"
+        className="text-natural-950"
+      >
+        {intl("tutor-settings.personal-info.title")}
+      </Typography>
+      <div className="flex items-center gap-8">
+        <div className="grow flex flex-col">
+          <Label>{intl("tutor-settings.personal-info.name")}</Label>
+          <Controller.Input
+            value={tutor.name || ""}
+            control={form.control}
```

Comment:
This is not right, the value should come from the from (the same applies to other fields). 

```suggestion
            value={form.watch('name') || ""}
```